projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -311,13 +318,13 @@ export class IgxChipComponent implements AfterViewInit {
      * ```
      */
     public set selected(newValue: boolean) {
-        const onSelectArgs = {
+        const onSelectArgs: IChipSelectEventArgs = {
             owner: this,
-            nextStatus: false,
+            selected: false,
             cancel: false
         };
         if (newValue && !this._selected) {
-            onSelectArgs.nextStatus = true;
+            onSelectArgs.selected = true;
```

Comment:
@alexalex972 This is a breaking change, so you should document it in the changelog as breaking change.